ui/app/templates/components/generated-item-list.hbs

```diff
@@ -64,11 +84,11 @@
                   Item.callMethod
                   "destroyRecord"
                   list.item
-                  (concat "Successfully deleted " (singularize itemType) " " list.item.id)
+                  (concat "Successfully deleted " (singularize itemType) " " list.item.id ".")
```

Comment:
Did you want to convert this to the new confirm style while you're in here? (totally fine if not!)

Reply:
i wanted to! i started down that path only to remember why i didn't do it the first time around: using the new style broke the markup :(